charts/agent-control/charts/agent-control-deployment/templates/rbac.yaml

```diff
@@ -30,19 +30,6 @@ rules:
       - get
       - list
       - watch
-  - apiGroups: [ "apps" ]
-    resources:
-      - deployments/scale
-    verbs:
-      - get
-      - patch
-      - update
-  - apiGroups: [ "" ]
-    resources: ["secrets"]
-    verbs:
-      - delete
```

Comment:
same but for GC?

Reply:
this was for the `{{ include "newrelic-agent-control.auth.secret.name" . }}` only

We have a few lines below the new condition:
```
  - apiGroups: [ "" ]
    resources: ["secrets"]
    verbs:
      - get
      - list
      - watch
      - create
      - delete
      - deletecollection
      - patch
      - update
```